libraries/rush-lib/src/utilities/Utilities.ts

```diff
@@ -302,119 +312,57 @@ export class Utilities {
    * Executes the command with the specified command-line parameters, and waits for it to complete.
    * The current directory will be set to the specified workingDirectory.
    */
-  public static executeCommand(options: IExecuteCommandOptions): void {
-    Utilities._executeCommandInternal(
-      options.command,
-      options.args,
-      options.workingDirectory,
-      options.suppressOutput ? undefined : [0, 1, 2],
-      options.environment,
-      options.keepEnvironment
-    );
-  }
-
-  /**
-   * Executes the command with the specified command-line parameters, and waits for it to complete.
-   * The current directory will be set to the specified workingDirectory.
-   *
-   * It's basically the same as executeCommand() except that it returns a Promise.
-   */
-  public static async executeCommandAndInspectOutputAsync(
-    options: IExecuteCommandOptions,
-    onStdoutStreamChunk?: (chunk: string) => string | void,
-    // eslint-disable-next-line @rushstack/no-new-null
-    onExit?: (exitCode: number | null, signal: NodeJS.Signals | null) => void
-  ): Promise<void> {
-    await Utilities._executeCommandAndInspectOutputInternalAsync(
-      options.command,
-      options.args,
-      options.workingDirectory,
-      options.suppressOutput ? undefined : onStdoutStreamChunk ? ['inherit', 'pipe', 'inherit'] : [0, 1, 2],
-      options.environment,
-      options.keepEnvironment,
+  public static async executeCommandAsync({
+    command,
+    args,
+    workingDirectory,
+    suppressOutput,
+    onStdoutStreamChunk,
+    environment,
+    keepEnvironment
+  }: IExecuteCommandOptions): Promise<void> {
+    await Utilities._executeCommandInternalAsync({
+      command,
+      args,
+      workingDirectory,
+      stdio: onStdoutStreamChunk ? ['inherit', 'pipe', 'inherit'] : suppressOutput ? undefined : [0, 1, 2],
```

Comment:
Does undefined prevent capturing stderr? It looks like it's used below in error scenarios. Seems like it would be valuable to capture regardless of the suppressOutput argument. Also, what does [0,1,2] do? Maybe add a comment?